geonode_mapstore_client/client/themes/geonode/less/_metadata.less

```diff
@@ -124,9 +124,16 @@
             padding: 0;
             .btn {
                 padding-left: 1.75rem;
+                border: none;
```

Comment:
we can use directly the `--ms-button-border-color: transparent;` under `.gn-metadata-list` selector without including additional rules. The css variables has been included in geonode style so they could overridden at css selector level